apps/sync/js/crypto/fxsyncwebcrypto.js

```diff
@@ -210,8 +212,8 @@ ${collectionName}`);
                                    payload.ciphertext)
           .then(recordArrayBuffer => {
         var recordObj;
-        const recordJSON = String.fromCharCode.apply(
-            null,
+        // The plaintext payload from FxSync is in UTF-8 encoding.
```

Comment:
Nit: I think you mean cleartext, not plaintext. Maybe a bit clearer if we just write:
// `recordArrayBuffer` contains UTF-8 JSON.